english/cowardice/cowardice.md

```diff
@@ -0,0 +1,5 @@
+You live a life in which the only reprieve from the deep dissatisfaction with your choices is the immeasurable pain which stems from 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.